### ISSUE TITLE
Issue: Added sites to the side bar should not open in new tabs - but in a dedicated side bar panel instead

### ISSUE THREAD

**Issue by Sa-Ja-Di** [brave/brave-browser#30480]
## Description 
To the side bar added sites are overwriting the present tab when opened  or can be opened in a new tab and are not being able to be opened in a dedicated side bar panel

## Steps to Reproduce
   1. open side bar
   2. add a page to the side bar
   3. click on the new side bar page icon

## Expected result:
Tabs added to the side bar shold open in a dedicated side bar panel, which should be able to expanded or minimized - wide of the panel should be decided by the user

## Reproduces how often: 
Always so far as the necessary feature is not added to the side bar

## Brave version (brave://version info)
<!--For installed build, please copy Brave, Revision and OS from brave://version and paste here. If 
1.52.102 Chromium: 113.0.5672.126 (Official Build) beta (64-bit)

**Comment by Arandhras** [comment]
The idea is nice, in this way you can basically keep two "lists" of tabs for different purposes, while having them all at hand.

**Comment by Sa-Ja-Di** [comment]
Yeah, dedicated side bar panels which can be size modified would be a good idea for added side bar items.

That way you can use 2 tabs at once, work and music playlist dedicated for example. That is why i love the progressive app side bar in Edge so much - for example i watch streams, one click to the side and Element opens.

in Brave.... tab is overwritten or it opens in an new tab... adding unnecessary extra steps.
